shop/src/main/java/ro/msg/learning/shop/controller/ProductOrderController.java

```diff
@@ -0,0 +1,55 @@
+package ro.msg.learning.shop.controller;
+
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.model.ProductOrder;
+import ro.msg.learning.shop.payroll.ProductOrderNotFoundException;
+import ro.msg.learning.shop.repository.IProductOrderRepository;
+
+
+import java.util.List;
+
+@RestController
+public class ProductOrderController {
```

Comment:
Take into consideration to refactor this controller too if needed (using previous instructions)